config/sources/families/include/rockchip64_common.inc

```diff
@@ -254,6 +254,11 @@ family_tweaks()
 		echo "auto lan0" >> $SDCARD/etc/network/interfaces
 		echo "	iface lan0 inet dhcp" >> $SDCARD/etc/network/interfaces
 
+	elif [[ $BOARD == helios64 ]]; then
+
+		chroot $SDCARD /bin/bash -c "apt-get -y -qq install mdadm i2c-tools"
+		chroot $SDCARD /bin/bash -c "apt-get -y -qq remove --auto-remove fake-hwclock>/dev/null 2>&1"
```

Comment:
Add to config/boards/helios64.wip with use of:

PACKAGE_LIST_BOARD | space delimited packages to be installed on this boards
PACKAGE_LIST_BOARD_REMOVE | space delimited packages to be removed
PACKAGE_LIST_DESKTOP_BOARD | space delimited packages to be installed on this boards desktop build
PACKAGE_LIST_DESKTOP_BOARD_REMOVE | space delimited packages to be removed

Reply:
ok, i'll move those lines and make use of the variables